### PR TITLE
LPS-107830 Avoiding XSS insertion in form configuration page

### PR DESCRIPTION
**Pull request by gabrielibson**

### CODE REVIEW

**Comment by gabrielibson** [comment]
ci:test:sf

**Comment by gabrielibson** [comment]
ci:test:relevant

**Comment by gabrielibson** [comment]
Related issue: https://issues.liferay.com/browse/LPS-107830

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/configuration.jsp [hunk 1]
```diff
@@ -83,7 +83,7 @@ DDMFormInstance selFormInstance = DDMFormInstanceServiceUtil.fetchFormInstance(f
 								sb.append("selectFormInstance('");
 								sb.append(formInstance.getFormInstanceId());
 								sb.append("','");
-								sb.append(HtmlUtil.escapeJS(formInstance.getName(locale)));
+								sb.append(HtmlUtil.escapeJS(HtmlUtil.escape(formInstance.getName(locale))));
```
Comment (leoadb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/configuration.jsp]: @gabrielibson isn't **HtmlUtil.escape** enough?

**Comment by leoadb** [comment]
sent to https://github.com/brianchandotcom/liferay-portal/pull/84806

**Comment by leoadb** [comment]
ci:close

**Pull request closed without merging**